node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
I haven’t compared the code side by side, but I think this should follow the pattern of the ps3 lib and overlay the values. I believe the ps3 lib also has unit tests for its loc functions.

However, it should follow the pattern of the ps3 lib *in spirit*, not verbatim. The node lib already has a function to import resource strings, we shouldn't add a second one. We should relax the constraint on the existing one.